gotext.go

```diff
@@ -142,6 +142,30 @@ func SetLibrary(lib string) {
 	loadStorage(true)
 }
 
+// GetGlobalLocalStorage is the locale storage getter for the package configuration.
+func GetGlobalLocalStorage() *Locale {
```

Comment:
Can we rename to GetStorage/SetStorage here? To maintain naming convention from other getter/setters.... unless you though of a reason to not do so.

Reply:
Sure, I found "Storage" to be a little bit too technical as this is the internal code, where what we are doing is really setting a locale. I’m fine changing it either way and understand the symmetry argument. As this is an advanced setting, I think people using it can dive into the upstream code :)